pkg/apis/core/validation/validation.go

```diff
@@ -3024,10 +3022,70 @@ func validateInitContainers(containers []core.Container, otherContainers []core.
 			allErrs = append(allErrs, field.Invalid(idxPath.Child("startupProbe"), ctr.StartupProbe, "must not be set for init containers"))
 		}
 	}
+
+	return allErrs
+}
+
+func validateContainer(ctr *core.Container, volumes map[string]core.VolumeSource, path *field.Path, opts PodValidationOptions) field.ErrorList {
```

Comment:
FYI, there's nothing new in `validateContainer()`, it's just the contents of the container loop in `validateContainers()`.